students/biegon_piotr/lesson_02_flow_control/the_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,16 @@
+print("The number of elements equal to the maximum\n")
+
+max1 = 0
```

Comment:
it's good that using `max` as name of variable was avoided (so no shadowing of built-ins).
However for sake of readability it would be good to use more elaborate names. In this scenario something like
`max_value` would fit better.